sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -0,0 +1,171 @@
+
+        public AzureMonitorTransmitter(AzureMonitorExporterOptions exporterOptions)
+        {
+            options = exporterOptions;
+            serviceRestClient = new ServiceRestClient(new ClientDiagnostics(options), HttpPipelineBuilder.Build(options));
+        }
+
+        internal async ValueTask<int> AddBatchActivityAsync(IEnumerable<Activity> batchActivity, CancellationToken cancellationToken)
+        {
+            if (cancellationToken.IsCancellationRequested)
+            {
+                return 0;
+            }
+
+            List<TelemetryEnvelope> telemetryItems = new List<TelemetryEnvelope>();
+            TelemetryEnvelope telemetryItem;
+
+            foreach (var activity in batchActivity)
+            {
+                telemetryItem = GeneratePartAEnvelope(activity);
+                telemetryItem.Data = GenerateTelemetryData(activity);
+                telemetryItems.Add(telemetryItem);
+            }
+
+            // TODO: Handle exception, check telemetryItems has items
+            var response = await this.serviceRestClient.TrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false);
+            return response.Value.ItemsAccepted.GetValueOrDefault();
+        }
+
+        private static TelemetryEnvelope GeneratePartAEnvelope(Activity activity)
+        {
+            // TODO: Get TelemetryEnvelope name changed in swagger
+            TelemetryEnvelope envelope = new TelemetryEnvelope(PartA_Name_Mapping[activity.GetTelemetryType()], activity.StartTimeUtc);
+            // TODO: Extract IKey from connectionstring
+            envelope.IKey = "IKey";
+            // TODO: Validate if Azure SDK has common function to generate role instance
+            envelope.Tags[ContextTagKeys.AiCloudRoleInstance.ToString()] = "testRoleInstance";
+
+            envelope.Tags[ContextTagKeys.AiOperationId.ToString()] = activity.TraceId.ToHexString();
+            if (activity.Parent != null)
+            {
+                envelope.Tags[ContextTagKeys.AiOperationParentId.ToString()] = activity.Parent.SpanId.ToHexString();
+            }
+
+            // TODO: "ai.location.ip"
+
+            envelope.Tags[ContextTagKeys.AiInternalSdkVersion.ToString()] = "dotnet5:ot0.4.0-beta:ext1.0.0-alpha.1"; // {language}{sdkVersion}:ot{OpenTelemetryVersion}:ext{ExporterVersion}
+
+            return envelope;
+        }
+
+        private MonitorBase GenerateTelemetryData(Activity activity)
+        {
+            MonitorBase telemetry = new MonitorBase();
+
+            var telemetryType = activity.GetTelemetryType();
+            telemetry.BaseType = Telemetry_Base_Type_Mapping[telemetryType];
+            string url = ProcessHttpTags(activity.Tags);
+
+            if (telemetryType == TelemetryType.Request)
+            {
+                var request = new RequestData(2, activity.Context.SpanId.ToHexString(), activity.Duration.ToString("c", CultureInfo.InvariantCulture), activity.GetStatus().IsOk, activity.GetStatusCode())
```

Comment:
RequestTelemetry.statuscode should be coming from the tag "http.status_code".
RequestTelemetry.success should be calculated based on the tag "http.status_code".
Don't use StatusCode in short :) 
This can be addressed in separate PRs.